public/v2/apps/shopware.json

```diff
@@ -0,0 +1,117 @@
+{
+    "captainVersion": "2",
+    "documentation": "See https://github.com/shyim/shopware-image for tags and other variables",
+    "displayName": "Shopware",
+    "description": "Shopware is a trendsetting ecommerce platform to power your online business. Our ecommerce solution offers the perfect combination of beauty & brains you need to build and customize a fully responsive online store.",
```

Comment:
I think the description is a bit too long. It'll mess up with the UI. I suggest removing `Our ecommerce solution...` and add it to `instructions.start`.